### PR TITLE
Update instructions for manual testing.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -215,16 +215,19 @@ flutter test test/ --update-goldens
 
 ## Manual Testing
 
-To run master version of DevTools with all experimental features enabled:
+If you do not have environment configured, but want to try newest DevTools with all experimental features enabled:
 
-1. Start DevTools
+1. Configure Dart or Flutter.
```
Comment (kenzieschmoll) [inline CONTRIBUTING.md:220]: if we are requiring them to configure flutter, then we don't need to say "If you do not have your environment configured". Can we change the above sentence to "If want to manually run DevTools with all experimental features enabled:"

Comment (kenzieschmoll) [reply CONTRIBUTING.md:220]: Also link to flutter getting started instructions

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -215,16 +215,19 @@ flutter test test/ --update-goldens
...
+
+2. Start DevTools:
 ```
 git clone git@github.com:flutter/devtools.git
+sudo ./devtools/tool/update_flutter_sdk.sh
 cd devtools/packages/devtools_app
-flutter run -d chrome --dart-define=enable_experiments=true
+../../flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
```
Comment (kenzieschmoll) [inline CONTRIBUTING.md:227]: Can we add instructions to the "# Development" section above for running DevTools with experiments? And including how to configure this flag in VS code would be nice.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -215,16 +223,19 @@ flutter test test/ --update-goldens
 
 ## Manual Testing
 
-To run master version of DevTools with all experimental features enabled:
+If want to explore DevTools with all experimental features enabled:
```
Comment (kenzieschmoll) [inline CONTRIBUTING.md:226]: nit: if you want

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -71,6 +71,14 @@ files for the DevTools Flutter app.
 
 To connect to your running application, paste the earlier copied observatory URL into the section "Connect to a Running App" in DevTools.
 
+To make VSCode's F5 having all DevTools` experiments enabled, add to the debugging configuration:
```
Comment (kenzieschmoll) [inline CONTRIBUTING.md:74]: nit: "To enable all experiments by default when you are running with VS Code, add the following to your debugging configuration:"

File: tool/update_flutter_sdk.sh [hunk 5]
```diff
@@ -4,14 +4,19 @@
...
 RELATIVE_PATH_TO_SCRIPT="${BASH_SOURCE[0]}"
 
 # The directory that this script is located in.
 TOOL_DIR=`dirname "${RELATIVE_PATH_TO_SCRIPT}"`
 
+cd `$TOOL_DIR`
+dart pub get
 REQUIRED_FLUTTER_VERSION=`dart $TOOL_DIR/bin/repo_tool.dart latest-flutter-candidate | tail -n 1`
-
+cd -
```
Comment (kenzieschmoll) [inline tool/update_flutter_sdk.sh:19]: what is 'cd -' for?

**Review by kenzieschmoll — APPROVED** [review]
lgtm with a couple comments and a question about `cd -`

File: tool/update_flutter_sdk.sh (hunk 5 above)
Comment (polina-c) [reply tool/update_flutter_sdk.sh:19]: `cd -` returns to previous path that was before using `cd somewhere`

**Pull request merged**